### PR TITLE
fix(ui): Display release filter dropdowns in a row

### PR DESCRIPTION
**Pull request by taylangocmen**
This updates the release page filter dropdowns to fit better in narrow widths.

DEMO:
https://user-images.githubusercontent.com/15015880/126244029-37cb61b4-3f7c-4452-9f70-408ca42e0036.mov

Fixes: [WOR-1055](https://getsentry.atlassian.net/browse/WOR-1055)

### CODE REVIEW

File: static/app/views/releases/list/index.tsx [hunk 1]
```diff
@@ -611,13 +613,26 @@ const AlertText = styled('div')`
...
+    flex-direction: row;
+  }
+`;
+
+const DropdownsWrapper = styled('div')`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  grid-gap: ${space(2)};
```
Comment (matejminar) [inline static/app/views/releases/list/index.tsx:635]: grid-gap still does not have great support for `display: flex`

File: static/app/views/releases/list/index.tsx [hunk 2]
```diff
@@ -611,12 +613,46 @@ const AlertText = styled('div')`
 `;
 
 const SortAndFilterWrapper = styled('div')`
-  display: inline-grid;
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
   grid-gap: ${space(2)};
```
Comment (matejminar) [inline static/app/views/releases/list/index.tsx:619]: We should not use gap on display:flex

File: static/app/views/releases/list/index.tsx [hunk 3]
```diff
@@ -611,12 +613,46 @@ const AlertText = styled('div')`
...
+   * Above this width search bar and controls can be on the same row */
+  @media (min-width: ${p => p.theme.breakpoints[2]}) {
+    flex-direction: row;
+  }
+`;
+
+const DropdownsWrapper = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: ${space(2)};
```
Comment (matejminar) [inline static/app/views/releases/list/index.tsx:636]: Also here, let's avoid using gap on display:flex

**Review by matejminar — COMMENTED** [review]
Unfortunately, gap on flex elements is still not supported on all major browsers.
This is safari:
image

File: static/app/views/releases/list/index.tsx (hunk 3 above)
Comment (taylangocmen) [reply static/app/views/releases/list/index.tsx]: 👍

File: static/app/views/releases/list/index.tsx (hunk 2 above)
Comment (taylangocmen) [reply static/app/views/releases/list/index.tsx]: :+1

**Review by scttcper — APPROVED** [review]

**Pull request merged**